include/obmc_console.hpp

```diff
@@ -117,8 +117,8 @@ inline void connectHandler(const boost::system::error_code& ec)
 inline void requestRoutes(App& app)
 {
     BMCWEB_ROUTE(app, "/console0")
-        .privileges({{"ConfigureComponents", "ConfigureManager"}})
         .websocket()
+        .privileges({{"ConfigureComponents", "ConfigureManager"}})
```

Comment:
This would be Admin and above? @joseph-reynolds You good with this? Or should readonly have access too?

Reply:
@joseph-reynolds please acknowledge so I make change based on that

Reply:
This set of privileges {ConfigureComponents,ConfigureManager} is only given to Administrator (and OemBMServiceAgent) users, and is not given to Operator or ReadOnly users.  Only Administrators should have acces to /console0 (host console), so this looks correct.